tests/smoke_tests/run_smoke_test.py

```diff
@@ -155,7 +155,7 @@ async def run_smoke_test(
     # times out after 20s of inactivity if it doesn't find the log message
     full_server_output = ""
     startup_messages = [
-        # printed by fedprox, apfl, basic_example, fedbn, fedper, fenda, fl_plus_local_ft and moon
+        # printed by fedprox, apfl, basic_example, fedbn, fedper, FENDA, fl_plus_local_ft and moon
```

Comment:
With the addition of new smoke test I wanted to make sure if we need update this comment.